app/models/concerns/orchestration/compute.rb

```diff
@@ -207,14 +207,20 @@ def find_image
   end
 
   def validate_compute_provisioning
-    return true unless image_build?
-    return true if ( compute_attributes.nil? or (compute_attributes[:image_id] || compute_attributes[:image_ref]).blank? )
-    img = find_image
-    if img
-      self.image = img
+    return true if compute_attributes.nil?
+    if image_build?
```

Comment:
I'll just leave it as is, merge and I'll try to remember this PR whenever we modify this code again :smile: